features/directory_structure.feature

```diff
@@ -67,6 +67,19 @@ Feature: Directory Structure
   configured `infer_spec_type_from_file_location!`, RSpec will automatically
   include the correct support functions for each type.
 
+  **Note:** `infer_spec_type_from_file_location!` affects only **the fixed list of folders**: controllers, helpers, jobs, mailers, models, requests, integration, api, routing, views, features.
+
+  To enable automatic metadata for other folder do the next:
```

Comment:
This reads weirdly, how about `To enable automatic metadata for other folders, use `RSpec::Configuration#define_derived_metadata`, for an example see below: